src/screens/Leaderboard/Leaderboard.tsx

```diff
@@ -200,7 +200,7 @@ function leaderboard(): JSX.Element {
                   <Avatar
                     key={_id + '1'}
                     containerStyle={styles.imageContainer}
-                    avatarStyle={styles.TableImage}
+                    avatarStyle={styles.TableImageSmall}
```

Comment:
Why was this changed?

Reply:
Changed to styles.TableImageSmall because the original size was causing the avatar images to overflow their space, affecting the layout.

![TableImageSmall](https://github.com/user-attachments/assets/23db44d2-95cf-4acd-9953-2c9b7bcb8bbb)